apps/docs/pages/guides/cli/seeding-your-database.mdx

```diff
@@ -49,11 +48,99 @@ You can use the `COPY` command to import CSV data into your database. For exampl
 ```sql supabase/seed.sql
 COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
 ```
+\*/}
+
+## Seed your database with Snaplet
+
+Snaplet generates seed data from a database schema and a definition of the intended values. For instance if you wanted to seed a typical blog application, with the following schema:
+
+![An example schema](/docs/img/guides/cli/snaplet-example-schema.png)
+
+{/*
+```sql schema.sql
+
+create table "User" (
+  id bigint not null primary key,
+  email text not null,
+  name text
+);
+
+create table "Comment" (
+  id bigint not null primary key,
+  text text not null,
+  "userId" bigint references "User" (id),
+  "postId" bigint references "Post" (id)
+);
+
+
+create table "Post" (
+  id bigint not null primary key,
+  title text not null,
+  content text,
+  "createdBy" bigint references "User" (id)
+);
+```
+\*/}
+
+And the intended values you want to generate:
+  * A `Post` with the title `"There is a lot of snow around here!"`
+  * The `Post.createdBy` user with an email address ending in `"@acme.org"`
+  * Three `Post.comments` from three different users.
+
+The definition of intended values would resemble:
+```ts snaplet.config.ts
+defineConfig({
+   generate: {
+      plan(snaplet) {
+        return snaplet.Post({
+          data: {
+            title: "There is a lot of snow around here!"
+            User: {
+                data: {
+                  email: copycat.email(seed, { domain: 'acme.org' })
+                }
+            },
+            Comment: {
+              count: 3,
+            }
+          }
+        })
+      }
+   }
+})
+```
+
+Running `npx snaplet generate` would produce these SQL statements:
```

Comment:
_Bikeshed_: Not sure if it matters enough to change, but its more that we're running these statements against the database than simply producing these statements (we're actually modify the database).

Reply:
We'll add a `--sql` flag to the command, it's just not in there at the moment.